### PR TITLE
Remove mocks from analyzer tests

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

**Comment by mfussenegger** [comment]
@seut please review

File: sql/src/test/java/io/crate/analyze/AlterTableAddColumnAnalyzerTest.java [hunk 1]
```diff
@@ -45,75 +39,49 @@
...
         expectedException.expect(IllegalArgumentException.class);
         expectedException.expectMessage("The table doc.users already has a column named name");
-        analyze("alter table users add column name string");
+        e.analyze("alter table users add column name string");
     }
 
     @Test
     public void testAddColumnToATableWithoutPrimaryKey() throws Exception {
-        AddColumnAnalyzedStatement analysis = (AddColumnAnalyzedStatement) analyze(
+        AddColumnAnalyzedStatement analysis = (AddColumnAnalyzedStatement)e.analyze(
```
Comment (seut) [inline sql/src/test/java/io/crate/analyze/AlterTableAddColumnAnalyzerTest.java]: pls check code format in this class (missing space).

File: sql/src/test/java/io/crate/analyze/CopyAnalyzerTest.java [hunk 2]
```diff
@@ -27,110 +27,77 @@
...
-    }
+
+public class CopyAnalyzerTest extends CrateUnitTest {
+
+    private SQLExecutor e = SQLExecutor.builder(new NoopClusterService()).enableDefaultTables().build();
 
     @Test
     public void testCopyFromExistingTable() throws Exception {
-        CopyFromAnalyzedStatement analysis = analyze("copy users from '/some/distant/file.ext'");
+        CopyFromAnalyzedStatement analysis =e.analyze("copy users from '/some/distant/file.ext'");
```
Comment (seut) [inline sql/src/test/java/io/crate/analyze/CopyAnalyzerTest.java]: check code format in this class also

File: sql/src/test/java/io/crate/analyze/CreateAlterPartitionedTableAnalyzerTest.java [hunk 3]
```diff
@@ -23,82 +23,52 @@
...
-            new OperatorModule())
-        );
-        return modules;
+public class CreateAlterPartitionedTableAnalyzerTest extends CrateUnitTest {
+
+    private SQLExecutor e;
+
+    @Before
+    public void init() throws Exception {
+        String  analyzerSettings = FulltextAnalyzerResolver.encodeSettings(
```
Comment (seut) [inline sql/src/test/java/io/crate/analyze/CreateAlterPartitionedTableAnalyzerTest.java]: remove space, pls check code format in this class.

**Pull request merged**